src/components/pagination/pagination.hbs

```diff
@@ -1,57 +1,90 @@
+
+    <div class="{{@root.prefix}}--pagination__right">
+      <div class="{{@root.prefix}}--select {{@root.prefix}}--select__page-number">
+        <select id="select-id-pagination-page" aria-label="select page number to view" class="{{@root.prefix}}--select-input" data-page-number-input>
+          {{#each pageNumberChoices}}
+            <option class="{{@root.prefix}}--select-option" value="{{value}}" {{#if selected}} selected{{/if}}>
+              {{label}} {{#if selected}} of {{totalPages}} {{/if}} 
+            </option> 
+          {{/each}} 
+        </select>
+        {{ carbon-icon 'ChevronDownGlyph' class=(add @root.prefix '--select__arrow')}}
+      </div>
+
+      <button class="{{@root.prefix}}--pagination__button {{@root.prefix}}--pagination__button--backward" data-page-backward aria-label="Backward button">
+        {{ carbon-icon 'CaretLeft24' class=(add @root.prefix '--pagination__nav-arrow')}}
+      </button>
+      <button class="{{@root.prefix}}--pagination__button {{@root.prefix}}--pagination__button--forward"
+        data-page-forward aria-label="Forward button">
+        {{ carbon-icon 'CaretRight24' class=(add @root.prefix '--pagination__nav-arrow')}}
+      </button>
+    </div>
+
   </div>
-  <div class="{{@root.prefix}}--pagination__right{{#is version "v2"}} {{@root.prefix}}--pagination--inline{{/is}}">
-    <span class="{{@root.prefix}}--pagination__text">
-      <span data-displayed-page-number>1</span> of
-      <span data-total-pages>4</span> pages</span>
-    <button class="{{@root.prefix}}--pagination__button {{@root.prefix}}--pagination__button--backward" data-page-backward aria-label="Backward button">
-      <svg class="{{@root.prefix}}--pagination__button-icon" width="7" height="12" viewBox="0 0 7 12">
-        <path fill-rule="nonzero" d="M1.45 6.002L7 11.27l-.685.726L0 6.003 6.315 0 7 .726z"/>
-      </svg>
-    </button>
-    <label for="page-number-input" class="{{@root.prefix}}--visually-hidden">Page number input</label>
-    {{#isnt version "v2"}}
-      <input id="page-number-input" type="text" class="{{@root.prefix}}--text-input" placeholder="0" value="1" data-page-number-input>
-    {{else}}
+  {{else}}
+  <div class="{{@root.prefix}}--pagination" data-pagination>
+    <div class="{{@root.prefix}}--pagination__left">
+      <div class="{{@root.prefix}}--select {{@root.prefix}}--select--inline">
+        <label for="select-id-pagination" id="select-id-pagination-label" class="{{@root.prefix}}--pagination__text">Items
+          per page: </label>
+        <select id="select-id-pagination" aria-label="select number of items per page" class="{{@root.prefix}}--select-input"
+          data-items-per-page>
+          {{#each itemsPerPageChoices}}
+          <option class="{{@root.prefix}}--select-option" value="{{value}}" {{#if selected}} selected{{/if}}>{{label}}
+            </option> {{/each}} </select> <svg class="{{@root.prefix}}--select__arrow" width="10" height="5" viewBox="0 0 10 5">
```

Comment:
Can use the `carbon-icon` helper for icons in the updated set (if needed). Docs: https://github.com/IBM/carbon-components/blob/master/docs/experimental.md#faq

Reply:
@joshblack we're only using svg in the v9 version, v10/X is using those helpers! is this proper implementation?

Reply:
My bad!